src/partials/footer.html

```diff
@@ -0,0 +1,37 @@
+<footer class="section-footer">
+  <!-- Style dodane jako przykład i propozycja implementacji bez dodawania bloków HTML bez znaczenia semantycznego -->
+  <style>
+    .social-media__link--github,
+    .social-media__link--telegram,
+    .social-media__link--facebook {
+      display: block;
+      width: 20px;
+      height: 20px;
+    }
+    .social-media__link--github::after {
+      content: url(./images/footer/github.svg);
+    }
+    .social-media__link--telegram::after {
+      content: url(./images/footer/telegram.svg);
+    }
+    .social-media__link--facebook::after {
+      content: url(./images/footer/facebook.svg);
+    }
+  </style>
+  <div class="container footer">
+    <div class="footer__header">
+      <p class="footer__header--upper">HOMEMADE ICE CREAM MADE</p>
```

Comment:
wpisujemy text normalnie bez capslocka,, zmienimy to potem css

Reply:
Poprawiono według komentarza